linkerd/install.go

```diff
@@ -48,17 +48,17 @@ var (
 )
 //to solve 
 type APIInfo struct { //to solve
-	TagName    string   `json:"tag_name,omitempty"`//to solve
-	PreRelease bool     `json:"prerelease,omitempty"` //to solve
-	Assets     []*Asset `json:"assets,omitempty"` //to solve
-} //to solve
+	TagName    string   `json:"tag_name,omitempty"`
+	PreRelease bool     `json:"prerelease,omitempty"` 
+	Assets     []*Asset `json:"assets,omitempty"` 
+} 
+//to solve
```

Comment:
Lets get rid of "to solve" and add a sensible comment here as well:
// Asset is used to store the individual asset data as part of a release